### ISSUE TITLE
Remove "View" and "Edit" tabs from annotator role view on edition Pages

### ISSUE THREAD

**Issue by daverett** [SHOREdevelopers/COVE#12]
_From @amandavisconti on February 15, 2016 19:10_

![screenshot_51](https://cloud.githubusercontent.com/assets/899033/13057990/da9696d6-d3ed-11e5-8a40-29895d123214.png)

_Copied from original issue: SHOREdevelopers/COVE#12_

**Comment by daverett** [comment]
I've removed permissions for the Annotator role to add and edit editions pages. That should be reserved for editors, correct? If we want annotators to be able to add and edit editions page but not see the tabs, I think we'll need to hide them in CSS.

**Comment by daverett** [comment]
Okay, I see that all the new users on the site have the annotator role, so it seems like there is no distinction between a "Lead Editor", as folks are titled in Dino's list, and annotators. I doubt this distinction would hold up, since annotators might be tasked with loading content, so I'm adding those permissions back. If this is right, then I would propose leaving the tabs there so that folks can easily make changes. It's also possible, though, to allow users to create but not edit content.

**Comment by daverett** [comment]
_From @amandavisconti on February 18, 2016 15:37_

Good point—I think people will largely want to both access the text (upload/edit changes) and annotate. On the other hand, editing the text after annotation has started can mess annotations up, so maybe we should just remove the permission to edit Editions Pages once the page has been uploaded, and add it back by request only?

**Comment by jamiefolsom** [comment]
@daverett Would you agree that the "annotator" role no longer makes sense in the Drupal COVE Editions Site, since all annotation work will occur inside of COVE Studio? If so, could the Instructor and/or Editor roles take its place?

**Comment by daverett** [comment]
Yes, except that the Editor/Instructor should have slightly elevated permissions compared to the previous annotator role. I would propose the following binaries:

Lead Editor -> Contributing editor 

AND, 

Instructor -> Student

This is to say that all roles can annotate but lead editors and instructors can also manage their teams. 

Maybe there is a better correspondence in AS that we can adopt?

D
